Entity/ProfessorSectionView.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy import Column, Integer, String, Enum, Text
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.dialects.mysql import SET
+import enum
+
+Base = declarative_base()
+
+
+class SectionType(enum.Enum):
+    activity = Act = 0
+    seminar = Sem = 1
+    independent = Ind = 2
+    lab = 3
+    lecture = Lec = 4
+
+
+class ProfessorSectionView(Base):
+    __tablename__ = "Professor_Teaches_Section_2"
```

Comment:
I believe `Professor_Teaches_Section_2` was a by product of me exploring the `CREATE VIEW` syntax in SQL.

Let's consider `Professor_Teaches_Section` to be the official `__tablename__`

Reply:
Fixed and tested